toolchain/check/scope_stack.cpp

```diff
@@ -147,20 +157,44 @@ auto ScopeStack::SetReturnedVarOrGetExisting(SemIR::InstId inst_id)
 auto ScopeStack::Suspend() -> SuspendedScope {
   CARBON_CHECK(!scope_stack_.empty()) << "No scope to suspend";
   SuspendedScope result = {.entry = scope_stack_.pop_back_val(),
-                           .suspended_lookups = {}};
+                           .suspended_lookups = {},
+                           .compile_time_bindings = {}};
   if (result.entry.scope_id.is_valid()) {
     non_lexical_scope_stack_.pop_back();
   }
   for (auto name_id : result.entry.names) {
     result.suspended_lookups.push_back(lexical_lookup_.Suspend(name_id));
   }
+  // Move any compile-time bindings into the suspended scope.
+  {
```

Comment:
FWIW, this extra scope mostly reads weird to me -- you might consider blank lines to separate logic. The separation of the comment is also an odd result.

Reply:
Done.